SerialPrograms/Source/PokemonSwSh/Programs/RNG/PokemonSwSh_Xoroshiro128Plus.cpp

```diff
@@ -0,0 +1,254 @@
+/*  Xoroshiro128+ and reverse
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
```

Comment:
Is there a place where you can link back to for this file? I assume you didn't generate this algorithm and table yourself.